src/request/import/index.html

```diff
@@ -61,6 +74,24 @@ <h1 data-i18n="import-heading-log-in">Log in</h1>
             </div>
         </div>
 
+        <!-- privacy agent -->
+        <div id="privacy" class="page">
+            <div class="page-header">
+                <a tabindex="0" class="page-header-back-button icon-back-arrow"></a>
+                <h1 data-i18n="import-heading-log-in">Log in</h1>
+            </div>
+            <div class="page-body">
+                <div class="grow"></div>
```

Comment:
This `grow` element should be removed to align the privacy agent to the top of the page body. The privacy agent contains a grow element itself between the text and the button, so that the button is always at the bottom of the page anyway.